app.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "payroll-system",
   "type": "app",
-  "defaultPage": "home",
+  "defaultPage": "employees",
```

Comment:
Can you revert this? We already set the payroll page as the default page.

Reply:
yep, noted sir.